src/Autofac/Builder/RegistrationBuilder{TLimit,TActivatorData,TRegistrationStyle}.cs

```diff
@@ -366,13 +366,28 @@ public RegistrationBuilder(Service defaultService, TActivatorData activatorData,
         /// <returns>A registration builder allowing further configuration of the component.</returns>
         public IRegistrationBuilder<TLimit, TActivatorData, TRegistrationStyle> PropertiesAutowired(PropertyWiringOptions wiringFlags)
```

Comment:
It will be a breaking change AFAIK at the binary level (it'll break anything compiled against the old version since the signature of the method will change), but not as far as API is concerned (the compiler will resolve the new method without issue). Since it's still pre-release of a major semantic version change (4.0), seems fine, but just wanted to make sure before refactoring.